packages/terra-dynamic-grid/tests/jest/DynamicGrid.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import DynamicGrid from '../../src/DynamicGrid';
+
+describe('DynamicGrid', () => {
+  const defaultRender = <DynamicGrid />;
```

Comment:
Children is a required prop of DynamicGrid. Does this render correctly? I am not seeing the snapshots.